### PR TITLE
Responsive cake webpage nader

### PR DESCRIPTION
**Pull request by naderakhgari**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,10 +7,56 @@
...
+        @import url('https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300&display=swap');
+    </style>
+
+
+
 </head>
 
 <body>
     <!-- Add your markup here -->
+    <nav id="navigation">
```
Comment (amcraethanan) [inline index.html]: Think about using the `<header>` element that was covered in last week's lesson - the nav would semantically make sense to live there in this case.

File: style.css [hunk 2]
```diff
@@ -1 +1,217 @@
...
+    height: 60px;
+}
+#logo{
+    margin-top: 0px;
+    padding-top: 0px;
+
+    height: 50px;
+    width: 120px;
+}
+a{
```
Comment (amcraethanan) [inline style.css]: what happens when you hover over hyperlinks? consider using a pseudo class to style `a` html elements as well

File: style.css [hunk 3]
```diff
@@ -1 +1,217 @@
...
+    margin-top: 10px;
+    margin-left: 0px;
+    width: 480px;
+    
+}
+#second-section{
+    /* height: 235px; */
+    width: 100%;
+    background-color:#f5f6fa;
+    display: flex;
```
Comment (amcraethanan) [inline style.css]: good job on remembering to use flexbox from last week's homework!

File: style.css [hunk 4]
```diff
@@ -1 +1,217 @@
...
+    flex-direction:column;
+    /* position: relative; */
+    margin-top: 10px;
+    align-items: center;
+}
+
+.second-section-img{
```
Comment (amcraethanan) [inline style.css]: Continue to favor using classes where possible in order to encourage re-usability of styles like this. Applying css using id selectors is mainly used when wanting to override class styles.

File: style.css [hunk 5]
```diff
@@ -1 +1,217 @@
...
+        margin-top: 0px;
+    }
+    #logo{
+        margin-top: 0px;
+        padding-top: 0px;
+        height: 90px;
+        width: 200px;
+    }
+    a{
+        text-decoration:none;
```
Comment (amcraethanan) [inline style.css]: if the styles are going to be the same regardless of screen size then they don't need to be written out again within the media query

File: index.html [hunk 6]
```diff
@@ -7,10 +7,58 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css?family=Anonymous+Pro|PT+Serif&display=swap');
+    </style>
+    <style>
+        @import url('https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300&display=swap');
+    </style>
```
Comment (jamesgorrie) [inline index.html]: _Minor:_ You could import these in the same style tag:
```
 <style>
    @import url('https://fonts.googleapis.com/css?family=Anonymous+Pro|PT+Serif&display=swap');
    @import url('https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300&display=swap');
</style>
```

File: style.css [hunk 7]
```diff
@@ -1 +1,217 @@
 /*  Add your styling here */
+
+#navigation{
```
[the remark this answers is not in the document]
Comment (jamesgorrie) [reply style.css]: What might be better is to use a classname to keep specificity low. [There is a great article here about it](https://css-tricks.com/strategies-keeping-css-specificity-low/).

File: style.css [hunk 8]
```diff
@@ -1 +1,218 @@
...
+    background-color:rgb(214, 235,215);
+    margin-top: 50px;
+    
+}
+#first-text{
+    margin-top: 0px;
+}
+.title{
+    font-family: 'Anonymous Pro', monospace;
+    font-family: 'PT Serif', serif;
```
Comment (jamesgorrie) [inline style.css]: Is there a reason for two `font-family` properties?

File: style.css [hunk 9]
```diff
@@ -1 +1,218 @@
...
+    text-align: center
+            
+}
+.text{
+    font-family: 'Open Sans Condensed', sans-serif;
+    font-size: 25px;
+    color: grey;
+    margin-top: 0px;
+    padding: 5px;
+    margin: 0px 30px 0px 30px;
```
Comment (jamesgorrie) [inline style.css]: _Minor:_ you can use the shorthand `margin: 0 30px`.

File: style.css (hunk 5 above)
Comment (jamesgorrie) [reply style.css]: [I also find this trick quite helpful](https://github.com/CodeYourFuture/responsive-cake-webpage/pull/6#discussion_r378096499), although it is completely personal preference.

**Pull request closed without merging**